docs/guides/admin/docs/configuration/stream-security.md

```diff
@@ -67,6 +67,11 @@ It is also possible to use one key for multiple URL prefixes:
     key.demoKeyThree.organization=mh_default_org
 
 
+A Java regular expression can be defined to identify URLs which shall not be signed:
+
+    exclude.url.pattern=.*/.*/unprotected/.*/.*
+
+
```

Comment:
We might consider mentioning that this is an "anchored pattern", i.e. the entire URL has to match, not just a substring.

The pattern `foo` does not match `http://localhost:8080/foo`, as I thought it would during my testing.

Reply:
Although "A Java regular expression..." implies that "foo" will not match "Hello foo".

Should I add some more documentation here?

Reply:
In the strict sense of the word "regular expression" you are right of course, but there are enough implementations around that would disagree; even Java's `Matcher` offers `lookingAt` in addition to `matches` which would make the regex `foo` match the string `foobar`.

So while in theory "regex" implies "has to match the entire string", in practice, and even in this case,  that is not a property inherent to regexes, but is rather determined by its use, and I think I would like to see something in the docs mentioning how it is used here. :thinking:

Reply:
ok, will add some more words here.

Reply:
@JulianKniephoff Updated documentation. Better?

Reply:
Yes, thanks! :smile: